### PR TITLE
Ported PHP to Python webserver

### PR DESCRIPTION
**Pull request by gmemstr**
Ported over the functionality of PHP files to Python webserver file. Currently runs on port 8080 but could be changed to port 80 if so desired (requires running under root in Linux), recommend using nginx as a reverse proxy (as I have personally preferred to do). 

Can properly select and return values from database as JSON, with some unfortunately sort of hacky methods of sanitizing input (should be just fine however). 

Chose Tornado over Flask (or Django) for future websockets support, which could lessen server strain when scaling. Also is non-blocking, a huge plus.

### CODE REVIEW

File: webserver.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+	def get(self, source):
+		resource = open(source).read()
+		self.set_header("Content-Type", 'text/css; charset="utf-8"')
+		self.write(resource)
+
+class DataHandler(tornado.web.RequestHandler):
+
+	def get(self, coin, change):
+		self.db = MySQLdb.connect('localhost', 'coingraphs', 'CGpassword', 'coingraphs')
```
Comment (nashe) [inline webserver.py]: Maybe all these values should be moved to a configuration file?

File: webserver.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		c  = self.db.cursor()
+
+		if coin != "eth" and coin != "btc": # Quick dirty workaround for sanitization
```
Comment (nashe) [inline webserver.py]: Same thing—maybe move it to a configuration file?

File: webserver.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		for row in data:
+			json_data.append(dict(zip(row_headers,row)))
+
+		self.write(json.dumps(json_data))
+
+
+def make_app():
+
+	settings = {
+		"debug": True
```
Comment (nashe) [inline webserver.py]: It's probably not a good thing to ship it with the debug mode enabled by default, it has security implications (in this case, mostly information disclosure).

File: webserver.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	}
+
+	# Build our routes
+	return tornado.web.Application([
+		(r"/", MainHandler),
+		(r"/assets/(.*)", ResourceHandler),
```
Comment (nashe) [inline webserver.py]: [StaticFileHandler](http://www.tornadoweb.org/en/stable/web.html#tornado.web.StaticFileHandler) should be used instead.

**Comment by derogab** [comment]
Good job @gmemstr !
I'm going to merge your pull request.

And then I'll edit the dependencies on README.md.
Thank you.

File: webserver.py (hunk 1 above)
Comment (nashe) [reply webserver.py]: See #4.

File: webserver.py (hunk 3 above)
Comment (nashe) [reply webserver.py]: See #4.

File: webserver.py (hunk 4 above)
Comment (nashe) [reply webserver.py]: See #5.

**Pull request merged**